topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,379 @@
+---
+layout: tutorial_hands_on
+title: Finding the muon stopping site with pymuon-suite in Galaxy 
+level: Introductory
+zenodo_link: https://zenodo.org/records/10219558
+questions:
+- "What are stopping sites and why is finding them important?"
+- "What computational techniques can be used to find stopping sites?"
+- "How can Galaxy help me with my analysis?"
+objectives:
+- "Understand why finding stopping sites is useful"
+- "Use pymuon-suite in Galaxy to find stopping sites"
+- "Understand the limitations of these tools and techniques"
+time_estimation: 1H
+key_points:
+- Galaxy supports multiple PyMuonSuite tools which can be used together to find muon stopping sites
+- The workflow for finding the stopping site involves three stages - structure generation, structure optimisation, and structure clustering
+- Within the UEP, the optimisation stage is based on analytically calculating the electrostatic forces applied to the muon, which can be performed via Galaxy.
+- The approach requires a CASTEP DFT calculation to be completed as a starting point
+contributions:
+  authorship:
+    - leandro-liborio 
+    - elichad
+    - patrick-austin
+  editing:
+    - leandro-liborio
+subtopic: muon-spectroscopy
+priority: 2
+abbreviations:
+  AIRSS: Ab Initio Random Structure Search
+  DFT: Density Functional Theory
+  UEP: Unperturbed Electrostatic Potential
+---
+
+
+# Introduction
```

Comment:
```suggestion
```

this section heading is unnecessary, we can generate a better article preview without it.